Doc/library/datetime.rst

```diff
@@ -646,8 +646,8 @@ Instance methods:
 
 .. method:: date.replace(year=self.year, month=self.month, day=self.day)
 
-   Return a date with the same value, except for those parameters given new
-   values by whichever keyword arguments are specified.
+   Return a new date with the same values, except with new values for
+   those parameters given by whichever keyword arguments are specified.
```

Comment:
Maybe `Returns a new :class:`date` with the same values, but with specified parameters updated.` would be an even better wording? Maybe even `date object`, but everything is an object in python, 

Also a note: lines 659-660 are not [visible](https://docs.python.org/3/library/datetime.html#datetime.date.replace) by some reason

Reply:
Thanks! I agree this wording sounds better.

Also I reworded lines 659-660 in a small way to get around whatever rst parsing barrier was preventing them from being shown.